client-course-schedulizer/package.json

```diff
@@ -56,14 +56,15 @@
     "@types/react-stickynode": "3.0.1",
     "@types/react-table": "^7.0.25",
     "@types/yup": "^0.29.9",
-    "isomorphic-fetch": "^3.0.0",
+    "fs": "0.0.1-security",
```

Comment:
No need for a change, just wondering what the reason is for a package change here?

Reply:
This was due to changing from pulling the test CSV files from github via a url (with isomorphic-fetch) to grabbing them locally using fs and another package which was already installed. I guess I'm not sure if isomorphic-fetch could have been used to do the same thing or not, but the first solution I found didn't.

Reply:
Cool!